.env.example.env

```diff
@@ -18,7 +18,7 @@ DB_ROOT_PASSWORD=drupal
 
```

Comment:
Hmm what happened here? :slightly_smiling_face: 

This should still be `.env.example`, don't rename this file.